common/app/routes/challenges/components/map/Challenge.jsx

```diff
@@ -144,6 +144,10 @@ export class Challenge extends PureComponent {
         challengeClassName
       );
     }
+    // do not render any challenges without a title.
+    if (!title) {
```

Comment:
@BerkeleyTrue 24 challenges have "undefined" passed as the title prop as found by console logging all the dashednames for undefined title challenges. including those four - the dashednames are in tact though and i dont see other occurrences of the "blank bullet" bug